var/spack/repos/builtin/packages/py-deeptools/package.py

```diff
@@ -31,13 +32,16 @@ class PyDeeptools(PythonPackage):
     depends_on("py-plotly@4.9:", when="@3.5.1:", type=("build", "run"))
     depends_on("py-plotly@2.0.0:", when="@:3.5.0", type=("build", "run"))
     depends_on("py-deeptoolsintervals@0.1.8:", type=("build", "run"))
+    depends_on("py-importlib-metadata", when="^python@:3.7", type=("build", "run"))
 
     def patch(self):
```

Comment:
We don't actually run any unit tests. I'm curious if this patch could be removed.

Reply:
Yep, as you say. I've removed it and tested that the oldest and latest versions install properly.